doc/release/README.tasks.rst

```diff
@@ -90,16 +108,18 @@ qualitatively speaking it will have a greater effect on programs which
 create more or shorter-lived tasks than on programs which create fewer
 or longer-lived ones.
 
-As described below in "Task Call Stacks", the execution-time default for
+As described in `Task Call Stacks`_, the execution-time default for
 stack overflow checking can be set by using the "--[no-]stack-checks"
```

Comment:
Do you think we should use code blocks for "--[no-]stack-checks"? I would lean towards yes, but don't feel strongly about it.